src/parser.mly

```diff
@@ -88,8 +88,8 @@ expr:
   | expr OR expr                   { Binop ($1, Or, $3) }
   | NOT expr                       { Unop (Not, $2) }
   | LPAREN expr RPAREN             { $2 }
+  | expr LSQUARE expr RSQUARE      { Elem ($1, $3) }
```

Comment:
what's an example of this usage? `expr[expr]` is like `5[5]`?

Reply:
see above